src/utils/label.ts

```diff
@@ -31,10 +31,13 @@ export interface LabelableComponent {
  * Exported for testing purposes only
  * @internal
  */
-export const labelClickEvent = "calciteInternalLabelClick";
+export const labelClickEvent = "labelClickEvent";
```

Comment:
Please follow event name conventions. These should use the `calciteInternal` prefix and do not need to end with `Event`.